openreview/venue_request/process/matchingProcess.py

```diff
@@ -0,0 +1,45 @@
+def process(client, note, invitation):
+
+    GROUP_PREFIX = ''
+    SUPPORT_GROUP = GROUP_PREFIX + '/Support'
+    request_form = client.get_note(note.forum)
+    conference = openreview.helpers.get_conference(client, note.forum)
+
+    build_conflicts=note.content.get('build_conflicts', None)
+
+    roles={
+        'reviewer': 'Reviewers',
+        'area chair': 'Area_Chairs',
+    }
+
+    matching_group = conference.get_id() + '/' + roles[note.content['matching_group'].strip()]
+    scores = note.content.get('affinity_scores', None)
+    file_name=None
+
+    if scores:
+        scores_attachment = client.get_attachment(id=note.id, field_name='affinity_scores')
+        with open('affinity_scores.csv','wb') as file:
+            file.write(scores_attachment)
```

Comment:
I would prefer to avoid writing the result into a file. I think the setup matching can read the content as a stream instead of a physical file.